src/useAudioPosition.ts

```diff
@@ -24,17 +23,17 @@ export const useAudioPosition = (
     config: UseAudioPositionConfig = {}
 ): AudioPosition => {
     const { highRefreshRate = false } = config
-    const { player, playing, stopped, duration } = useContext(context)!
+    const { player, playing, stopped, duration } = useContext(playerContext)!
+    const { position, setPosition } = useContext(positionContext)!
```

Comment:
if you adjust the type of the position context (see above comment) then you should be able to drop `!` as the value should never be null